docs/tutorial/06-proxy.md

```diff
@@ -0,0 +1,50 @@
+# Step 6: Proxy
```

Comment:
I feel like we shouldn't have a page for this.

This has no action for the user to take and is tangent to the process

The previous page should have a note pointing out they pass in `frontendProject` to setup the frontend project to work with the node application.

Reply:
I actually had it as one step originally, but I felt like "Creating a node app" and "Connect frontend to Node"  was easier to follow. So I wanted to make sure they run serve for backend first.